File: java/organizations-micronaut-java/src/main/java/com/talentwunder/controller/OrganizationController.java
package com.talentwunder.controller;

import com.talentwunder.dto.OrganizationDTO;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

import java.util.List;

@Controller("/organizations")
public class OrganizationController {

    @Get("/")
    List<OrganizationDTO> get() {
        return List.of(new OrganizationDTO("75d19288-6c17-4c62-9337-8d67de0ac6af", "test-organization"));
    }
}

File: kotlin/organizations-micronaut-kotlin/src/main/kotlin/com/talentwunder/controller/OrganizationController.kt
package com.talentwunder.controller

import com.talentwunder.dto.OrganizationDTO
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/organizations")
class OrganizationController {

    @Get("/")
    fun get(): List<OrganizationDTO> {
        return listOf(OrganizationDTO("eb3791d3-825b-4fd2-a5ac-ad8e17c529a4", "test-organization"))
    }
}
